internal/ingress/controller/nginx.go

```diff
@@ -116,8 +114,7 @@ func NewNGINXController(config *Configuration, fs file.Filesystem) *NGINXControl
 
 		fileSystem: fs,
 
-		// create an empty configuration.
-		runningConfig: &ingress.Configuration{},
+		runningConfig: new(ingress.Configuration),
```

Comment:
Is `new` a more (in comparison to `&`) standard way of initializing objects in Go? Is it doing something different than `&` in this specific case?

Reply:
`new` is pretty much equivalent to `&Type{}`, it allocates memory for the given Type. I like to use for initialization if zero values, but it's mostly sugar.